src/methods/receiver.md

```diff
@@ -13,3 +13,5 @@ are other possible receivers for a method:
   transmitted.
 * No receiver: this becomes a static method on the struct. Typically used to
   create constructors which are called `new` by convention.
+
+There are even more types, e.g., `Box<Self>`.
```

Comment:
Probably rephrase this to make it clear where these types come from, and that they're receiver types, e.g.

"Beyond variants on `self`, there are also [special](https://doc.rust-lang.org/reference/special-types-and-traits.html) wrapper types allowed to be receiver types, such as `Box<Self>`.

Reply:
Thanks, that's a nice way to put it, I updated the PR to say that.